spec/docker-compose.yml

```diff
@@ -6,15 +6,6 @@ services:
     ports:
       - "20022:22"
 
-  puppet_5_node:
-    build:
-      context: .
-      args:
-        PUPPET_COLLECTION: puppet5
-    container_name: puppet_5_node
-    ports:
-      - "20023:22"
```

Comment:
I think in the future if anyone asks why we don't have `20023` we should say it's an unlucky number.

Reply:
And if they ask why, just respond "think about it". ;P